### ISSUE TITLE
Pixelized (nonsmooth) segmentation output 

### ISSUE THREAD

**Issue by EvaBr** [EvaBr/AnatomyNets#1]
Before the adjustments to Losses.py  and  use of LogSoftmax instead of Softmax, the network didn't learn much (not even to segment everything as background, despite large class imbalance). But at least the output segmentation looked smooth:
![smooth](https://user-images.githubusercontent.com/6882207/113752540-603e5980-970d-11eb-953b-96d6c8acb79a.png)

After changes, it looks suspiciously pixelated:
![pixelated](https://user-images.githubusercontent.com/6882207/113752556-63d1e080-970d-11eb-8668-676ff1af4223.png)

However, I have not been able to revert to the smooth version. 
Issues: why is it pixelated at all?  
Epsilon and Strategy change in Dice is NOT the issue (reverting doesn't change the smoothess back).  Going back from LogSoftmax -> Softmax also doesn't change anything. Tried with different amounts of training data, also no change.
